### PR TITLE
Implement API route to return a specific Skill

### LINKED ISSUE #14: Implement API route to return a specific Skill
On the `/resume/skill` route, you should

- [ ] Return a specific Skill when a GET request is made with an index as an input. The Skill should be in a JSON format.

### PR DESCRIPTION
**Pull request by emilyllim**
Closes #14 

Update pytest file to include a test for retrieving a specific Skill.

Update GET method to return a specific Skill based on the index provided.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -82,7 +82,14 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        index = request.args.get("index")
+        # if an index is not provided, return all Skills
+        if index == None:
+            return jsonify(data["skill"])
+        index = int(index)
```
Comment (fissoreg) [inline app.py:89]: What if the index is not an integer?

Comment (emilyllim) [reply app.py]: Just added a digit check for the index

**Comment by fissoreg** [comment]
Thank you @emilyllim, LGTM! If you can resolve the conflicts, we can merge this!

**Comment by emilyllim** [comment]
> Thank you @emilyllim, LGTM! If you can resolve the conflicts, we can merge this!

Thank you! I just resolved the conflicts. The pull request is ready to be merged.

**Pull request merged**